File: Smartsense.Mobile.Android/app/src/main/java/com/smartsense/covid/api/model/responses/QuarantineStatusResponse.java
package com.smartsense.covid.api.model.responses;

import java.util.List;

public class QuarantineStatusResponse extends BaseApiResponse{

    private boolean quarantineStatus;

    public boolean isQuarantineStatus() {
        return quarantineStatus;
    }

    public void setQuarantineStatus(boolean quarantineStatus) {
        this.quarantineStatus = quarantineStatus;
    }

    public QuarantineStatusResponse(List<String> errors) {
        super(errors);
    }
}

File: Smartsense.Mobile.iOS/covid/Api/Responses/QuarantineStatusResponse.swift
//
//  UserLoginResponse.swift
//  covid
//
//  Created by SmartSense on 8.06.2021.
//

import Foundation

struct QuarantineStatusResponse: BaseApiResponse, Codable{
    let userId: Int64?
    let quarantineStatus: Bool?
    var code: String?
    var message: String?
    var errors: [String?]?
    
    init(userId: Int64?, quarantineStatus: Bool?, code: String? = nil, message: String? = nil, errors: [String?]? = nil) {
        self.userId = userId
        self.quarantineStatus = quarantineStatus
        self.code = code
        self.message = message
        self.errors = errors
    }
    
}
